monai/handlers/mlflow_handler.py

```diff
@@ -70,6 +75,13 @@ class MLFlowHandler:
         state_attributes: expected attributes from `engine.state`, if provided, will extract them
             when epoch completed.
         tag_name: when iteration output is a scalar, `tag_name` is used to track, defaults to `'Loss'`.
+        experiment_name: name for an experiment, defaults to `default_experiment`.
+        run_name: name for run in an experiment, defaults to `test_run`.
+        experiment_param: a dict recording parameters which will not change through whole experiment,
+            like torch version, cuda version and so on.
+        artifacts: paths to images that need to be recorded after a whole run.
+        optimizer_param_names: parameters' name in optimizer that need to be record during runing,
+            defaults to `["lr"]`.
```

Comment:
I think we need a more general way to track `optimizer` parameters, especially when there are multiple optimizers and we may also need to track `lr_scheduler`.
@wyli @ericspod Do you experts have any suggestions?

Thanks in advance.